stl/inc/atomic

```diff
@@ -998,6 +1115,41 @@ struct _Atomic_storage<_Ty, 16> { // lock-free using 16-byte intrinsics
         return _Result != 0;
     }
 
+#if _HAS_CXX20
+    void wait(const _Ty _Expected, const memory_order _Order = memory_order_seq_cst) const noexcept {
+        const auto _Storage_ptr       = _STD addressof(_Storage);
+        _Int128 _Expected_bytes = _Atomic_reinterpret_as<_Int128>(_Expected);
+
+        for (;;) {
+            _Int128 Observed_bytes = _Atomic_reinterpret_as<_Int128>(load(_Order));
+            if (_Expected_bytes._Low != _Observed_bytes._Low || _Expected_bytes._High != _Observed_bytes._High) {
+#if _CMPXCHG_MASK_OUT_PADDING_BITS
+                if constexpr (_Might_have_non_value_bits<decltype(_Storage._Storage)>) {
+                    _Storage_for<_Ty> _Mask{_Form_mask};
+                    const _Value_type _Mask_val = _Atomic_reinterpret_as<_Int128>(_Mask._Ref());
+
+                    if (((_Expected_bytes._Low ^ _Observed_bytes._Low) & _Mask_val._Low) == 0 && ((_Expected_bytes._High ^ _Observed_bytes._High) & _Mask_val._High) == 0) {

```

Comment:
This exceeds 120 columns. If you wrap it, will clang-format preserve that wrapping? (It sometimes acts strangely in the presence of preprocessor directives.) If clang-format insists on unwrapping it and exceeding the configured 120 limit, then I recommend using an empty comment to force wrapping, like this:
```
cute fluffy kittens //
beginning on a new line
```